ledger/participant-integration-api/src/test/suite/scala/platform/store/dao/AppendOnlySchemaMigrationSpec.scala

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao
+
+import com.codahale.metrics.MetricRegistry
+import com.daml.ledger.api.domain.{LedgerId, ParticipantId}
+import com.daml.ledger.api.testing.utils.AkkaBeforeAndAfterAll
+import com.daml.ledger.resources.{Resource, ResourceContext, ResourceOwner}
+import com.daml.lf.data.Ref
+import com.daml.lf.engine.{Engine, ValueEnricher}
+import com.daml.logging.LoggingContext
+import com.daml.logging.LoggingContext.newLoggingContext
+import com.daml.metrics.Metrics
+import com.daml.platform.configuration.ServerRole
+import com.daml.platform.store.dao.events.LfValueTranslation
+import com.daml.platform.store.{DbType, FlywayMigrations}
+import com.daml.testing.postgresql.PostgresResource
+import org.scalatest.Succeeded
+import org.scalatest.wordspec.AsyncWordSpec
+
+class AppendOnlySchemaMigrationSpec extends AsyncWordSpec with AkkaBeforeAndAfterAll {
+  "Postgres flyway migration" should {
+
+    // TODO append-only: this test is disabled because the current schema uses the
+    // PARTITION keyword which is not supported in the current Postgres version
+    "successfully migrate to append-only schema if the database is empty" ignore {
```

Comment:
I would even say this is not necessarily needed. Feel free to remove if you see fit.